instrument/src/main/java/edu/berkeley/cs/jqf/instrument/mutation/MCLCache.java

```diff
@@ -0,0 +1,38 @@
+package edu.berkeley.cs.jqf.instrument.mutation;
+
+import java.net.URL;
+
+/**
+ * A lazy cache of {@link MutationClassLoader}s, one for each instance, with the
+ * same {@code URL} list and parent {@code ClassLoader}
+ */
+public class MCLCache {
```

Comment:
This is not really a "cache" since there's always a 1:1 mapping between mutation instances and mutation classloaders. It's not like there's a fast path and slow path. Can you rename this to something more meaningful, such as simply `MutationClassLoaders`?